### PR TITLE
tijolo: 0.7.4 -> 0.8.1

### PR DESCRIPTION
**Pull request by sund3RRR**
## Description of changes
Update version
https://github.com/hugopl/tijolo/releases/tag/v0.8.0

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by sund3RRR** [comment]
@MaxwellDupre @bjornfor could you review this PR please?

File: pkgs/applications/editors/tijolo/make.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
++++ b/Makefile	2024-04-20 11:43:19.950786808 +0300
+@@ -6,8 +6,10 @@
+ 	shards build --release -Dpreview_mt $(CR_FLAGS) -s
+ 
+ configure:
+-	shards install
+-	./bin/gi-crystal
++	mkdir bin/
++	cd lib/gi-crystal && shards build -Dpreview_mt --release --no-debug
++	cd ../..
```
Comment (bjornfor) [inline pkgs/applications/editors/tijolo/make.patch:11]: I think each line in a Make target is run by a separate shell, so this "cd ../.." is basically a NOP.

**Review by bjornfor — APPROVED** [review]
LGTM. Didn't test.

**Comment by bjornfor** [comment]
@sund3RRR: There are conflicts that must be resolved.

**Comment by sund3RRR** [comment]
Do I need to squash a merge commit?

**Comment by bjornfor** [comment]
> Do I need to squash a merge commit?

git fetch origin
git rebase origin/master
(Resolve conflicts)
git push -f mygithubremote

**Comment by sund3RRR** [comment]
GG I broke the branch

**Comment by sund3RRR** [comment]
I think it is because I've already done a rebase to update the branch with new libadwaita.
Probably I can fix it using rebase to commit in reflog history.

**Comment by sund3RRR** [comment]
Okay I fixed the branch, but I don't know how to squash this merge commit because another commit from another PR appears.

**Pull request closed without merging**